vector/src/main/res/values/strings.xml

```diff
@@ -515,8 +515,8 @@
     <string name="room_one_user_is_typing">%s is typing…</string>
     <string name="room_two_users_are_typing">%1$s &#038; %2$s are typing…</string>
     <string name="room_many_users_are_typing">%1$s &#038; %2$s &#038; others are typing…</string>
-    <string name="room_message_placeholder_encrypted">Send an encrypted message…</string>
-    <string name="room_message_placeholder_not_encrypted">Send a message (unencrypted)…</string>
+    <string name="room_message_placeholder_encrypted">Message…</string>
+    <string name="room_message_placeholder_not_encrypted">Message…</string>
```

Comment:
Please create new string resource(s?) for riotx

Reply:
added new resource